src/reference/asciidoc/whats-new.adoc

```diff
@@ -1,180 +1,20 @@
 [[whats-new]]
 === What's New
 
-==== Changes in 2.0 Since 1.7
-
-===== CachingConnectionFactory
-
-Starting with _version 2.0.2_, the `RabbitTemplate` can be configured to use a different connection to that used by listener containers.
-This is to avoid deadlocked consumers when producers are blocked for any reason.
-See <<separate-connection>> for more information.
+==== Changes in 2.1 Since 2.1
```

Comment:
Typo: Since `2.0`, I guess :smile: 

I think we need to do similar refactoring for SI as well